### PR TITLE
Add vimscript version

### PR DESCRIPTION
**Pull request by robjperez**

### CODE REVIEW

File: vimscript/stalinsort.vim [hunk 1]
```diff
@@ -0,0 +1,15 @@
+function! s:StalinSort(list)
+  if empty(a:list)
+    return []
+  endif
+  let l:return = [a:list[0]]
+  for x in a:list
+    if x > l:return[-1]
```
Comment (gustavo-depaula) [inline vimscript/stalinsort.vim]: what is the result of a list with repeated numbers like `1 3 3 2 4`?

Comment (robjperez) [reply vimscript/stalinsort.vim]: It actually outputs [1, 3, 4]

Comment (gustavo-depaula) [reply vimscript/stalinsort.vim]: can you change it to output `1 3 3 4`?

Comment (robjperez) [reply vimscript/stalinsort.vim]: Done in the last commit :)

**Pull request merged**